cfn-resources/cloud-backup-restore-jobs/cmd/resource/resource.go

```diff
@@ -0,0 +1,341 @@
+package resource
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/handler"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
+	progressevents "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	publicKey                        = "ApiKeys.PublicKey"
+	privateKey                       = "ApiKeys.PrivateKey"
+	projectID                        = "ProjectId"
+	snapshotID                       = "SnapshotId"
+	clusterName                      = "ClusterName"
+	id                               = "Id"
+	errorCreateMongoClient           = "Error - Create MongoDB Client- Details: %+v"
+	errorCreateCloudBackupRestoreJob = "Error - Create Cloud Backup Restore snapshot for Snapshot(%s)- Details: %+v"
+	errorReadCloudBackUpRestoreJob   = "Error - Read Restore Job with id(%s)"
+	automated                        = "automated"
+	download                         = "download"
+)
```

Comment:
Can we utilize "github.com/mongodb/mongodbatlas-cloudformation-resources/util/constants" for consts?

Reply:
sure we will move this

Reply:
This is fixed.Please check now